components/raftstore/src/store/config.rs

```diff
@@ -172,6 +175,13 @@ pub struct Config {
     #[online_config(hidden)]
     pub apply_yield_duration: ReadableDuration,
 
+    #[serde(with = "engine_config::perf_level_serde")]
+    #[online_config(skip)]
+    pub perf_level: PerfLevel,
+
+    #[online_config(skip)]
+    pub trigger_write_size: ReadableSize,
```

Comment:
Should add comment describing what it is. And etc/config-template.toml should be updated too.

Reply:
Updated. I change its name to `raft_write_size_limit`.